sqlitebrowser/umbrel-app.yml

```
@@ -0,0 +1,45 @@
+manifestVersion: 1
+id: sqlitebrowser
+name: SQLite Browser
+tagline: https://sqlitebrowser.org/
```

For the tagline, what do you think about:

```suggestion
tagline: A database browser for SQLite
```

Based on their about page: https://sqlitebrowser.org/about/